zerver/tests/test_hooks.py

```diff
@@ -1215,6 +1395,7 @@ def build_webhook_url(self):
         return self.URL_TEMPLATE.format(api_key=api_key, stream=self.STREAM_NAME, data_template=url_data)
```

Comment:
I think `URL_TEMPLATE` should be prefixed with a `u`, since otherwise it's difficult to check whether this function returns `text_type` or `str`.